src/aatoolbox/config/countryconfig.py

```diff
@@ -26,33 +26,73 @@ class CodABConfig(BaseModel):
     """
 
     hdx_dataset_name: str
-    layer_base_name: str  # TODO: validate that it has {admin_level}
+    layer_base_name: str
     admin_level_max: int
```

Comment:
This is more of a general question about configs in general, but the `admin_level_max` is based solely on file availability, is that right? And `layer_base_name` requires `{admin_level}` in it. Can we just automatically detect the max admin level when it's instantiated? Are we already checking that the files exist at some point? Then could it just be detected then?

Just wondering aloud here, if it makes sense, could create an issue and tackle it later on down the road.

Reply:
Yeah, really good point, I will make an issue!

Reply:
#104